resources/js/Components/ComponentsPraktikans/MainLanding.jsx

```diff
@@ -6,7 +6,7 @@ export default function MainLanding() {
             <div className="text-center">
                 <h1 className="text-deepForestGreen font-bold text-3xl">Welcome To</h1>
                 <h1 className="text-deepForestGreen font-bold text-[55px]">Daskom Laboratory</h1>
-                <Link href={route('login')}>
+                <Link href="login">
```

Comment:
```suggestion
                <Link href="/login">
```
shouldnt this be `/login`?